front/src/modules/ui/board/components/EntityBoard.tsx

```diff
@@ -33,6 +34,17 @@ import { BoardOptions } from '../types/BoardOptions';
 
 import { EntityBoardColumn } from './EntityBoardColumn';
 
+export type EntityBoardProps = {
+  boardOptions: BoardOptions;
+  onColumnAdd?: (boardColumn: BoardColumnDefinition) => void;
+  onColumnDelete?: (boardColumnId: string) => void;
+  onEditColumnTitle: (columnId: string, title: string, color: string) => void;
+  scopeContext: Context<string | null>;
+} & Pick<
+  BoardHeaderProps<PipelineProgresses_Order_By>,
```

Comment:
OK but we need to refactor sorts later. it feels weird ot user BoardHeaderProps here